File: engine/model/src/main/scala/net/server/channel/packet/guild/ChangeGuildRankPacket.scala
package net.server.channel.packet.guild

class ChangeGuildRankPacket(private var _type: Byte, private var _playerId: Int, private var _rank: Byte) extends BaseGuildOperationPacket(_type = _type) {
  def playerId: Int = _playerId

  def rank: Byte = _rank
}

File: engine/engine-model/src/main/java/net/server/channel/packet/guild/ChangeGuildRankPacket.java
package net.server.channel.packet.guild;

public class ChangeGuildRankPacket extends BaseGuildOperationPacket {
   private final Integer playerId;

   private final Byte rank;

   public ChangeGuildRankPacket(Byte theType, Integer playerId, Byte rank) {
      super(theType);
      this.playerId = playerId;
      this.rank = rank;
   }

   public Integer playerId() {
      return playerId;
   }

   public Byte rank() {
      return rank;
   }
}
